Rectangle.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOP_Lab1
+{
+    public class Rectangle : IFigure
+    {
+        protected int X1 { get; set; }
+        protected int X2 { get; set; }
+        protected int Y1 { get; set; }
+        protected int Y2 { get; set; }
+
+        public Rectangle(int x1, int x2, int y1, int y2)
+        {
+            if (x1 < 0 || x2 <= 0 || y1 <= 0 || y2 <= 0)
+            {
+                x1 = Math.Abs(x1);
+                x2 = Math.Abs(x2);
+                y1 = Math.Abs(y1);
+                y2 = Math.Abs(y2);
+            }
+
+            int temp;
+            
+            if (x1 > x2)
+            {
+                temp = x1;
```

Comment:
you should create private SwapCoordinates(x,y) method here for the readability

Reply:
oh debagger what am i wasting my youth on